contracts/src/CrossChainApplications/examples/NativeTokenBridge/tests/NativeTokenDestinationTests.t.sol

```diff
@@ -233,6 +238,7 @@ contract NativeTokenDestinationTest is NativeTokenBridgeTest {
 
         new NativeTokenDestination(
             MOCK_TELEPORTER_REGISTRY_ADDRESS,
+            address(this),
```

Comment:
I actually have the reverse question, why `msg.sender` in the cases above instead of `address(this`) 😅 

My understanding is that `address(this)` is the unit test contract itself, meaning later on in the test we could directly make calls as the owner. I'm not sure what the `msg.sender` value is within the unit test functions (presumably some default address that coordinates the test runs, or is mocked out). I'd recommend not using `msg.sender` here either way since it's not clear what the address will be, and it may change if forge changes the behavior. Instead, I'd recommend using a default address that we define as a constant, and can mock if needed.

Reply:
I meant to use `address(this)`, instead now I added a `DEFAULT_OWNER_ADDRESS` constant. A bit of a pain in the owner upgradeable tests since the tests need to `vm.prank` the owner address to test calls, but outside of that contract don't think it'll be used in too many unit tests

Reply:
Is there a reason this constructor call still uses `address(this)` while the other test constructor calls use `_DEFAULT_OWNER_ADDRESS`?